### PR TITLE
Add inheritance graphs to docs

### PR DESCRIPTION
**Pull request by dstansby**
Fixes https://github.com/matplotlib/napari-matplotlib/issues/169. Here's what it looks like rendered locally for me:

Screenshot 2023-06-15 at 10 01 01

### CODE REVIEW

**Comment by samcunliffe** [comment]
Yeah, do you think it's worthwile? 
Since #168 there is now some relatively interesting class structure.

Also: custom plot users will need to inherit from... `SingleAxisWidget` now, right?

**Comment by dstansby** [comment]
I think it's worth it, especially if https://github.com/matplotlib/napari-matplotlib/pull/167 happens and it gets even more *interesting*

File: docs/conf.py [hunk 1]
```diff
@@ -63,8 +64,11 @@
 # Can't work out how to link this properley using intersphinx and the PyQT6 docs.
```
Comment (samcunliffe) [inline docs/conf.py:64]:
```suggestion
# Can't work out how to link this properly using intersphinx and the PyQT6 docs.
```

**Comment by samcunliffe** [comment]
Do we need @ruaridhg because: co-authored commit?

**Comment by dstansby** [comment]
I think I hit the merge buton, it's just waiting for tests.

**Comment by dstansby** [comment]
@samcunliffe can you try giving this a fresh approve?

**Review by samcunliffe — APPROVED** [review]

**Pull request merged**